salt/modules/lxc.py

```diff
@@ -2508,7 +2591,11 @@ def bootstrap(name,
     # default set here as we cannot set them
     # in def as it can come from a chain of procedures.
     if bootstrap_args:
-        bootstrap_args = '{0} -c {{0}}'.format(bootstrap_args)
+        # custom bootstrap args can be totally customized, and user could
```

Comment:
but why? the only thing for now here is to inject here  the random config directory placeholder, why complicating which 1 or 2 __named__ placeholders, this complicates the whole thing up